opentelekomcloud/customize_diffs.go

```diff
@@ -38,3 +40,32 @@ func validateRDSv3Version(argumentName string) schema.CustomizeDiffFunc {
 		return nil
 	}
 }
+
+func validateCCEClusterNetwork(d *schema.ResourceDiff, meta interface{}) error {
+	config, ok := meta.(*Config)
+	if !ok {
+		return fmt.Errorf("error retreiving configuration: can't convert %v to Config", meta)
+	}
+	vpcClient, err := config.networkingV1Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("error creating opentelekomcloud CCE Client: %s", err)
+	}
+
+	vpcID := d.Get("vpc_id").(string)
+	if len(vpcID) != 0 {
+		err = vpcs.Get(vpcClient, vpcID).Err
+		if err != nil {
```

Comment:
You can use single-line expression:
`if err := vpcs.Get(vpcClient, vpcID).Err; if err != nil {`